package.json

```diff
@@ -132,6 +132,7 @@
     "fsevents": "^2.1.3"
   },
   "peerDependencies": {
+    "react": ">= 16",
```

Comment:
Why do we need this?

Reply:
simply, follow CRA's changes: https://github.com/facebook/create-react-app/pull/9872/files
but without this, 'enact serve -f' works well

Reply:
The 9872 PR has not landed yet... Why should we take this in advance?

Reply:
https://github.com/facebook/create-react-app/pull/9872
: this PR has several changes, and not only for updating plugin ~0.4.3 (add "react" in peerDependencies is not necessary)
i will remove this